package.json

```diff
@@ -14,51 +16,64 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "@embroider/util": "^0.39.1 || ^0.40.0 || ^0.41.0",
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-htmlbars": "^5.1.0"
+    "@embroider/util": ">= 0.39.1 < 2.0.0-alpha.1",
```

Comment:
@knownasilya I'm sorry to get back on this, `@embroider/util` is now `1.0.1` so we should be able to do just `"@embroider/util": "^1.0.0"`